WAL/WAL/Screen/Setting/View/MyInfoTableViewCell.swift

```diff
@@ -28,11 +28,8 @@ final class MyInfoTableViewCell: BaseTableViewCell {
     let socialTypeLabel = UILabel().then {
         $0.font = WALFont.body9.font
         $0.textColor = .gray100
-        if UserDefaultsHelper.standard.social == SocialType.kakao.rawValue {
-            $0.text = SocialType.kakao.login
-        } else if UserDefaultsHelper.standard.social == SocialType.apple.rawValue {
-            $0.text = SocialType.apple.login
-        }
+        $0.text = UserDefaultsHelper.standard.social == SocialType.KAKAO.rawValue ?
+        SocialType.KAKAO.login : SocialType.APPLE.login
```

Comment:
```suggestion
        let buttonType = SocialType(rawValue: UserDefaultsHelper.standard.social) ?? .KAKAO
        $0.text = buttonType.login 
```
제가 쓴게 잘 돌아가는 코드인지 모르겠지만 이런 식으로도 쓸 수 있을 것 같아여 ~

Reply:
깔끔하네여 관련 부분 저렇게 수정해볼게여